integreat_cms/cms/views/utils/translation_status.py

```diff
@@ -0,0 +1,36 @@
+"""
+This file contains the helper function to change the translation file
+"""
+
+from django.contrib import messages
+from django.utils.translation import gettext_lazy as _
+
+
+def change_translation_status(request, selected_content, language_slug, desired_status):
+    """
+    Helper function to change the translation status
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param selected_content: The current queryset
+    :type selected_content: ~integreat_cms.cms.models.abstract_content_model.ContentQuerySet
+
+    :param language_slug: The language slug of the current language
+    :type language_slug: str
+
+    :param desired_status: The desired status
+    :type desired_status: str
+    """
+
+    if request.user.is_superuser or request.user.is_staff:
+        for content in selected_content:
+            translation = content.get_translation(language_slug)
+            translation.status = desired_status
+            translation.save()
+            messages.success(
+                request,
+                _("The status of {} was successfully changed to {}").format(
+                    translation, desired_status.lower()
+                ),
+            )
```

Comment:
This is my suggestion to the translation problem:

```suggestion
    status = {
        "PUBLIC": "Published", 
        "DRAFT": "Draft"
    }

    if request.user.is_superuser or request.user.is_staff:
        for content in selected_content:
            translation = content.get_translation(language_slug)
            translation.status = desired_status
            translation.save()
            messages.success(
                request,
                _("The status of \"{}\" was successfully changed to \"{}\"").format(
                    translation, _(status[desired_status])
                ),
            )
```

Reply:
Oh, and we can maybe add an error message for else-case. I know those options are not shown for non staff users but...... 😅